src/azul/security/verifier.py

```diff
@@ -0,0 +1,111 @@
+# The code is based on dss.util.security.
+from base64 import urlsafe_b64decode
+import logging
+from urllib.parse import urlparse
+
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.backends import default_backend
+import jwt
+from jwt.exceptions import DecodeError, PyJWTError
+import requests
+
+from azul import config
+
+logger = logging.getLogger(__name__)
+
+allowed_algorithms = ('RS256',)
+gserviceaccount_domain = "iam.gserviceaccount.com"
+
+# Shared HTTP session
+session = requests.Session()
```

Comment:
No side-effects at module loading/import time, please. Please fix in a follow-up PR.